src/modules/previewpane/powerpreview/settings.cpp

```diff
@@ -104,16 +209,34 @@ namespace PowerPreviewSettings
 		FileExplorerPreviewSettings(
 			false,
             GET_RESOURCE_STRING(IDS_PREVPANE_SVG_BOOL_TOGGLE_CONTROLL),
-            GET_RESOURCE_STRING(IDS_PREVPANE_SVG_SETTINGS_DESCRIPTION)){}
+            GET_RESOURCE_STRING(IDS_PREVPANE_SVG_SETTINGS_DESCRIPTION),
+			L"{ddee2b8a-6807-48a6-bb20-2338174ff779}",
```

Comment:
Can we store the GUID's at one place and reuse them in C# and C++ project ?

Reply:
There is a way to do this:
1. Create a static StringsTable class that contains all the constant hard coded strings and provide access methods to it. The CPP project will load the DLL and run the access methods to get the GUIDS. The cons about this is that we will have to be (1) moving files during build, (2) Will have to change all the hardcoded strings (in attributes) in the PreviewHanlders code to use the ones in static classes, (3) this means more code and changes each time a preview handler is added.

2. The way im currently doing, only one place needs to be updated to add a preview handler and that is the GUID.

Reply:
The only cons i can think off for the second one is maintaining consistency in the GUIDS given to the CPP project and the ones used in the C# project.

Reply:
But the same GUID is required in C# as well. So we have to make sure that GUID at these two places remain same.

Reply:
Yes, the only way I know to do that is in (1) above which is out of the scope of the PR to make changes to the Preview Handlers

Reply:
I think we should put a task for this